src/components/FilterList/FilterHelper.ts

```diff
@@ -63,17 +63,17 @@ export const filtersMatchURL = (filterTypes: FilterType[], filters: ActiveFilter
   // This can probably be improved and/or simplified?
   const fromFilters: Map<string, string[]> = new Map<string, string[]>();
   filters.filters.forEach(activeFilter => {
-    const existingValue = fromFilters.get(activeFilter.category) || [];
-    fromFilters.set(activeFilter.category, existingValue.concat(activeFilter.value));
+    const existingValue = fromFilters.get(activeFilter.id) || [];
+    fromFilters.set(activeFilter.id, existingValue.concat(activeFilter.value));
   });
 
   const fromURL: Map<string, string[]> = new Map<string, string[]>();
   const urlParams = new URLSearchParams(history.location.search);
   filterTypes.forEach(filter => {
     const values = urlParams.getAll(filter.id);
     if (values.length > 0) {
-      const existing = fromURL.get(filter.title) || [];
-      fromURL.set(filter.title, existing.concat(values));
+      const existing = fromURL.get(filter.id) || [];
+      fromURL.set(filter.id, existing.concat(values));
     }
   });
```

Comment:
Yep this is the problem that I commented you about that filter used title instead I'd